src/main/java/mcp/mobius/betterbarrels/BetterBarrels.java

```diff
@@ -129,7 +129,7 @@ public void preInit(FMLPreInitializationEvent event) {
 			
 			allowOreDictUnification = config.getBoolean("allowOreDictUnification", Configuration.CATEGORY_GENERAL, true, "If true, Jabba will try unificate 'ingot' 'ore' 'dust' and 'nugget' using oredict");
 
-			renderDistance = config.getFloat("renderDistance", Configuration.CATEGORY_GENERAL, 16f, 0f, 10000f, "Render Distance (square) for stack and text on barrel.");
+			renderDistance = config.getFloat("renderDistance", Configuration.CATEGORY_GENERAL, 10000f, 0f, 10000f, "Render Distance (square) for stack and text on barrel.");
```

Comment:
Can we change the text for the config to describe that it is actually cubed (not squared) distance.

Reply:
No. Check the code again. It is indeed squared.

Reply:
Apologies in that case. I might have misunderstood something.